### PR TITLE
read your meter

### PR DESCRIPTION
**Pull request by smhgit**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @smhgit, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;wheels-custom-integrations#35).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: components/read_your_meter.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "read_your_meter",
+  "owner": ["smhgit"],
+  "manifest": "https://https://raw.githubusercontent.com/eyalcha/read_your_meter/master/custom_components/read_your_meter/manifest.json",
```
Comment (frenck) [inline components/read_your_meter.json:4]: That URL seems wrong...?

Comment (eyalcha) [reply components/read_your_meter.json]: typo, https was duplicated. fixed

Comment (eyalcha) [reply components/read_your_meter.json]: @frenck are there any changes I need to do? I am not sure as it shows "Changes requested" but on the other hand all checks have passed.

Comment (frenck) [reply components/read_your_meter.json]: I was awaiting some builds and changes upstream. This PR has potentially issue with building on our end. Hence I was waiting for everything to be clear at our end).

**Pull request merged**